libpod/runtime_cstorage.go

```diff
@@ -106,18 +106,18 @@ func (r *Runtime) removeStorageContainer(idOrName string, force bool) error {
 				logrus.Infof("Storage for container %s already removed", ctr.ID)
 				return nil
 			}
-			return errors.Wrapf(err, "error looking up container %q mounts", idOrName)
+			logrus.Errorf("error checking if container %q is mounted, attempting to delete: %v", idOrName, err)
```

Comment:
```suggestion
			logrus.Errorf("Error checking if container %q is mounted, attempting to delete: %v", idOrName, err)
```

Reply:
Since we have lots of different cases for logrus.Errorf, I would rather just get this merged and then work on a separate PR to fix all of the logrus.Errorf calls.  We need to decide if they should all be UpperCase or lower case.

Reply:
The convention for logs is always upper case (errors lower case), so I think the decision has been made on that already.